doc/lua_api.txt

```diff
@@ -5240,6 +5240,9 @@ This is basically a reference to a C++ `ServerActiveObject`
     * in first person view
     * in third person view (max. values `{x=-10/10,y=-10,15,z=-5/5}`)
 * `get_eye_offset()`: returns `offset_first` and `offset_third`
+* `send_mapblock(blockpos)`:
```

Comment:
I can't find much documentation to `blockpos`. The only thing I found is:
https://github.com/minetest/minetest/blob/77961aa73d1947a2eabaa832319b7d73db6d13c2/doc/lua_api.txt#L4094-L4095
Which isn't really helpful.
How should you know where a block begins? (Eg. is there a block from x = 0 to x = 15 or x = -2 to x = 13 or whatever?)
It would be useful to have helper functions for this.

Reply:
'blockpos' needs clearly documenting and explaining certainly. Will make an issue to remind myself.

Reply:
Documented in #8376

Reply:
`mapblock` would be less confusing here.